services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/client_api.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import traceback
+from typing import Dict, Optional
+
+import httpx
+from fastapi import FastAPI
+
+from ...core.settings import DynamicSidecarSettings
+from ...models.schemas.dynamic_services import MonitorData
+from .errors import MonitorException
+
+logger = logging.getLogger(__name__)
+
+
+def get_url(dynamic_sidecar_endpoint: str, postfix: str) -> str:
+    """formats and returns an url for the request"""
+    url = f"{dynamic_sidecar_endpoint}{postfix}"
+    return url
+
+
+def log_httpx_http_error(url: str, method: str, formatted_traceback: str) -> None:
+    # mainly used to debug issues with the API
+    logging.debug(
+        (
+            "%s -> %s generated:\n %s\nThe above logs can safely "
+            "be ignored, except when debugging an issue "
+            "regarding the dynamic-sidecar"
+        ),
+        method,
+        url,
+        formatted_traceback,
+    )
+
+
+class DynamicSidecarClient:
```

Comment:
Again, another important modules with **zero** test coverage :-(

TIP: use ``respx`` to emulate the dynamic sidecar behavior and set a couple of usual scenarios as a base

Reply:
this was already done, please refer to the comment above to see the current coverage.